src/Bonsai.ML.Torch/Buffer.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.ComponentModel;
+using System.Linq;
+using System.Reactive.Linq;
+using TorchSharp;
+
+namespace Bonsai.ML.Torch;
+
+/// <summary>
+/// This operator collects incoming tensors into a buffer and concatenates them along the first dimension.
```

Comment:
```suggestion
/// Represents an operator that gathers incoming tensors into zero or more tensors by concatenating them along the first dimension.
```

Reviewing wording for style consistency with other packages. Using "gather into a tensor" form, to make clear the output type is a tensor.